src/opnsense/mvc/app/views/layouts/default.volt

```diff
@@ -301,6 +301,13 @@
     <!-- bootstrap script -->
     <script src="{{ cache_safe('/ui/js/bootstrap.min.js') }}"></script>
     <script src="{{ cache_safe('/ui/js/bootstrap-select.min.js') }}"></script>
+    <script>
+        /* patch translations into selectpicker library */
+        jQuery.fn.selectpicker.Constructor.DEFAULTS.noneSelectedText = "{{ lang._('Nothing selected') }}";
+        jQuery.fn.selectpicker.Constructor.DEFAULTS.noneResultsText = "{{ lang._('No results matched {0}') }}";
+        jQuery.fn.selectpicker.Constructor.DEFAULTS.selectAllText = "{{ lang._('Select All') }}";
+        jQuery.fn.selectpicker.Constructor.DEFAULTS.deselectAllText = "{{ lang._('Deselect All') }}";
```

Comment:
you could simplify that with Object.assign(target, source).

You can find some background information at https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
BTW: I do already have translations for Bootgrid in this file:

https://github.com/opnsense/core/blob/6d959b603919dc28e6de50f06f5548eea232e1d2/src/opnsense/mvc/app/views/layouts/default.volt#L185-L195

It would be good if you could put this block after mine.